### ISSUE TITLE
如果用keras，需要怎么设置？keras的backend是tensorflow

### ISSUE THREAD

**Issue by alyato** [EncodeTS/TensorFlow_Center_Loss#1]
谢谢。
能够用keras，但是具体的backend作为更改，不是很会。
请问如果在keras中添加，需要怎么操作。

**Comment by EncodeTS** [comment]
印象中，在Keras中自定义操作符的时候，没办法方便地取到该batch数据的label，因此也就没办法构造Center Loss。如果其他人有解决方案，可以告知我一下。

**Comment by alyato** [comment]
@EncodeTS 
[https://github.com/shamangary/Keras-MNIST-center-loss-with-visualization](url)
该作者写了一个，我也用了，可是在自己的数据集上效果不好。

**Comment by EncodeTS** [comment]
你是用在什么任务上？

**Comment by alyato** [comment]
疾病分类。给一张人脸，分类一种疾病。
